siliconcompiler/schema/schema_cfg.py

```diff
@@ -2112,6 +2116,27 @@ def schema_option(cfg):
             may load multiple flows and libraries.
             """)
 
+    scparam(cfg, ['option','pdk'],
+            sctype='str',
+            scope='job',
+            shorthelp="PDK target",
+            switch="-option_pdk <str>",
+            example=["cli: -option_pdk freepdk45",
```

Comment:
other options don't have option prefix, should be `-pdk`

Reply:
NIce catch, fixed.